src/Sets/HalfSpace.jl

```diff
@@ -594,33 +632,8 @@ Finally, the returned set is the half-space with normal vector `[a1, …, an]` a
 displacement `b`.
 """
 function HalfSpace(expr::Operation, vars::Union{<:Operation, <:Vector{Operation}}=get_variables(expr); N::Type{<:Real}=Float64)
-
-    # find sense and normalize
-    if expr.op == <
-        a, b = expr.args
-        sexpr = simplify(a - b)
-
-    elseif expr.op == >
-        a, b = expr.args
-        sexpr = simplify(b - a)
-
-    elseif (expr.op == |) && (expr.args[1].op == <)
-        a, b = expr.args[1].args
-        sexpr = simplify(a - b)
-
-    elseif (expr.op == |) && (expr.args[2].op == <)
-        a, b = expr.args[2].args
-        sexpr = simplify(a - b)
-
-    elseif (expr.op == |) && (expr.args[1].op == >)
-        a, b = expr.args[1].args
-        sexpr = simplify(b - a)
-
-    elseif (expr.op == |) && (expr.args[2].op == >)
-        a, b = expr.args[2].args
-        sexpr = simplify(b - a)
-
-    else
+    (valid, sexpr) = _is_halfspace(expr)
```

Comment:
```suggestion
    valid, sexpr = _is_halfspace(expr)
```
Just a suggestion, we can also keep the previous version.